docs/configure-arquillian.md

```diff
@@ -0,0 +1,95 @@
+
+**Notice:** Be aware of false positive test results when using certain versions of TestNG with Arquillian. More information can be found [here](https://github.com/WASdev/arquillian-liberty-dependencies/blob/master/docs/testng-false-positives.md).
+
+#### Automatic generation of `arquillian.xml`
+
+For both Arquillian Liberty Managed and Remote containers, the `configure-arquillian` goal can automatically generate and configure the `arquillian.xml` that usually exists in the `src/test/resources` directory, eliminating the need for the user to manually create the file in projects with only a single Arquillian container. 
+
+Furthermore, when using the Arquillian Liberty Managed container, `configure-arquillian` will also automatically specify the three required configuration parameters: `wlpHome`, `serverName`, and `httpPort`. These are set to values specified in the `liberty-maven-plugin`. 
+
+The `configure-arquillian` goal will perform configuration for the Liberty Managed or Remote container based on which container exists on the classpath. Arquillian does not allow for more than one container on the classpath. In the event of neither container existing on the classpath, the `configure-arquillian` goal will default to providing configuration for the Liberty Managed container. 
+
+**Common configuration parameters for the Arquillian Liberty Managed and Remote configurations:**
+
+| Property | Type | Description | Default |
+-----------| ------------ | ------- | ------- |
+| `arquillianProperties` | Dictionary | Used to set key/value pairs of configuration parameters in `arquillian.xml`. | **Managed:** A dictionary containing values for `wlpHome`, `serverName`, and `httpPort` as specified in the `liberty-maven-plugin`.<br>**Remote:** An empty dictionary when using the Arquillian Liberty Remote container. |
+| `skipIfArquillianXmlExists` | Boolean | Skips the `configure-arquillian` goal if `arquillian.xml` already exists in the `target` directory. | False |
+
+TODO: Add link to JBoss documentation of configuration parameters. These links might be outdated due to container name change...
+
+https://docs.jboss.org/author/display/ARQ/WLP+V8.5+-+Managed
+
+https://docs.jboss.org/author/display/ARQ/WLP+V8.5+-+Remote
+
+#### Usage Example
+
+Specify the following `liberty-maven-plugin` configuration in `pom.xml`:
+
+```
+<plugin>
+	<groupId>net.wasdev.wlp.maven.plugins</groupId>
+	<artifactId>liberty-maven-plugin</artifactId>
+	<version>2.1.2</version>
+	<extensions>true</extensions>
+	<!-- Specify configuration, executions for liberty-maven-plugin -->
+	<configuration>
+		<serverName>LibertyProjectServer</serverName>
+		<assemblyArtifact>
+			<groupId>com.ibm.websphere.appserver.runtime</groupId>
+			<artifactId>wlp-webProfile7</artifactId>
+			<version>17.0.0.3</version>
```

Comment:
Why 17.0.0.3?

Reply:
Will update

Reply:
Can [17.0.0.4,) be used here? (to mean x>=17.0.0.4)